grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -54,7 +54,16 @@ public ComputerResult compute(MapReduce mapReduce) {
         try {
             return getComputer().mapReduce(mapReduce).submit().get();
         } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException(e);
+            throw asRuntimeException(e);
+        }
+    }
+
+    private RuntimeException asRuntimeException(Throwable throwable) {
```

Comment:
The issue is like this:
- The graph computers rethrow all exceptions by wrapping them in an "ExecutionException".
- Sometimes the inner exceptions can be interpreted further down the line based on their type (e.g. ConceptException).
- If we rethrow everything as plain `RuntimeException`, the shell will print "unexpected error", when the error might really be "there is no path" or similar.

It was my idea, but I agree it's weird and hope there's a better way!